### PR TITLE
Added task "Create Circle with OOP"

### PR DESCRIPTION
**Pull request by VolhaVaitulevich**
Link: https://edabit.com/challenge/Hgb38yhWGwJCMHbRQ

### CODE REVIEW

File: exercises/classes/makeCircle/index.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class Circle {
+	radius: number;
+	constructor(radius: number) {
+		this.radius = radius;
+	}
+
+	getArea(): number {
+		return Math.PI * Math.pow(this.radius, 2);
+	}
+	getperimeter(): number {
```
Comment (siarhei-papkou) [inline exercises/classes/makeCircle/index.ts]: getPerimeter

File: exercises/classes/makeCircle/index.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+class Circle {
```
Comment (siarhei-papkou) [inline exercises/classes/makeCircle/index.ts]: Let's export all our solutions.
`export class Circle`

**Review by siarhei-papkou — APPROVED** [review]

File: exercises/classes/makeCircle/index.ts (hunk 1 above)
Comment (VolhaVaitulevich) [reply exercises/classes/makeCircle/index.ts]: Updated in a new commit

File: exercises/classes/makeCircle/index.ts (hunk 2 above)
Comment (VolhaVaitulevich) [reply exercises/classes/makeCircle/index.ts]: Updated in a new commit

**Review by siarhei-papkou — APPROVED** [review]

**Pull request merged**